Viktoriia/TicTacToe/main.cpp

```diff
@@ -0,0 +1,142 @@
+#include<iostream>
+using namespace std;
+
+void ShowImage(char mas[][3], int size);
+void Cross(char mas[][3], int size);
+void Zero(char mas[][3], int size);
+bool FieldFilling(char mas[][3], int size, char possCross, char possZero);
+bool CheckWin(char mas[][3], int size);
+bool EndGame(char mas[][3], int size);
+
+int main()
+{
+    const int HEIGHT = 3;
+    const int WIDTH = 3;
+    char arr[HEIGHT][WIDTH] = { { '1','2','3' }, { '4', '5','6' }, { '7','8','9' } };
+    ShowImage(arr, 3);
+    Cross(arr,3);
+	return 0;
+}
+
+void ShowImage(char mas[][3], int size) // ���� ��������� ����
+{
+        for (int i = 0; i < size; i++)
+        {
+            for (int j = 0; j < size; j++)
+            {
+                cout << "    " << mas[i][j] << "     |    " << mas[i][j + 1] << "     |    " << mas[i][j + 2] << "    \n" << "----------|----------|---------\n";
+                break;
+            }
+        }
+}
+
+void Cross(char mas[][3], int size) 
+{
+    char possCross;
+    cout << " Player 1(X), please enter the number of the position where you want to put a cross: ";
+    cin >> possCross;
+    if (!FieldFilling(mas, 3, possCross, '15')) // ���� �� ������� ������� ����� ��� ����� �����
+    {
+        cout << "Please try again!\n ";
+        Cross(mas, 3);
+    }
+    
+    ShowImage(mas, 3);
+    if (CheckWin(mas, 3)) //�������� �� � �������� ��������
+    {
+        cout << " Player 1(X) you are Winner!" << endl;
+        EndGame(mas, 3);
+    }
+    else if (!EndGame(mas,3)) //�������� �� �� ������ �������� � ��� �
+    {
+        Zero(mas, 3);
+    }
+    else
+        cout << "GAME OVER!!!!!!" << endl;
```

Comment:
First time you used braces `{}` for one line statement, second time - not. Please choose one style and use it. I am ok with both ways but always using braces for one line statements is preferable, because it is less error prune